app/javascript/chat/chat.jsx

```diff
@@ -130,7 +131,9 @@ export default class Chat extends Component {
 
     setupObserver(this.observerCallback);
 
-    this.subscribePusher(`private-message-notifications--${appName}-${currentUserId}`);
+    this.subscribePusher(
```

Comment:
I think code climate is failing because of this extra line added by the prettier

Reply:
That's fine, we can override it on out end.